scout/server/blueprints/cases/templates/cases/gene_panel.html

```diff
@@ -98,60 +98,72 @@ <h6 class="ms-3 mt-3"><span class="fas fa-clipboard-list"></span>&nbsp;HPO gene
     </div>
 
     <!-- scrollable div containing dynamic gene list -->
-    <div class="mt-3 ms-3" style="max-height:200px; overflow-y: scroll; min-height: 100px;">
-      {% if case.dynamic_gene_list %}
-        <i>
-        {% for hpo_gene in case.dynamic_gene_list %}
-          {{ hpo_gene.hgnc_symbol }}&nbsp;&nbsp;
-        {% endfor %}
-        </i>
-      {% else %}
-        <span class="text-muted">No gene list generated</span>
-      {% endif %}
-    </div>
+      <div class="mt-3 ms-3" style="max-height:200px; overflow-y: scroll; min-height: 100px;">
+        {% if case.dynamic_gene_list %}
+          <form id="dynamicGenes" method="POST" action="{{ url_for('cases.phenotypes_actions', institute_id=institute._id, case_name=case.display_name) }}" >
+            <i>
+            {% for hpo_gene in case.dynamic_gene_list %}
+              <input type="checkbox" name="dynamicGene" id="dynamicGene" value="{{hpo_gene.hgnc_id}}">
+              <input type="hidden" name="action" value="REMOVEGENES">
+              {{ hpo_gene.hgnc_symbol }}
+              &nbsp;&nbsp;
+            {% endfor %}
+            </i>
+          </form>
+        {% else %}
+          <span class="text-muted">No gene list generated</span>
+        {% endif %}
+      </div>
```

Comment:
How about adding a warning either indicator badge or just text like the above if the panel has been modified, like if the case hpo_ids list `dynamic_panel_phenotypes` is empty I suppose - though adding a more explicit var could be nice to simplify future maintenance. That would alleviate my primary concern with the manual remove approach: that one user removes a few genes after generating the HPO panel and just forgets about it, thinking the whole panel is checked.

Reply:
I see your point. I'll see what I can do!

Reply:
I've added some code to display a red `(edited)` whenever the dynamic gene list is edited by adding or removing genes

<img width="626" alt="image" src="https://user-images.githubusercontent.com/28093618/185370441-62f05c44-232e-4cf0-a9c7-da3d3d3c5d10.png">

Reply:
I just thought about what you said about it not being retroactive. You could also check if `dynamic_panel_phenotypes` is empty but there are genes on the panel. Afaik it should always be filled with stuff if  and only if the panel is unedited.

Reply:
In the situation you describe is quite obvious that the panel has been modified. There are also these situations dynamic panels created in the past, over which you have no control:
- Dynamic gene panels with genes added
- Dynamic gene panels with 2 phenotypes where people changed one of the 2 phenotypes so the genes change over time but you don't know (you still have two phenotypes)

I suggest let's leave it as it is since it would just complicate the code without really adding much.

Reply:
> In the situation you describe is quite obvious that the panel has been modified. There are also these situations for dynamic panels created in the past, over which you have no control:
> 
> * Dynamic gene panels with genes added

Right, but the add gene endpoint does clear the `dynamic_panel_phenotypes` so that should be covered.

> * Dynamic gene panels with 2 phenotypes where people changed one of the 2 phenotypes so the genes change over time but you don't know (you still have two phenotypes)

True, though that does not really affect the dynamic panel list: as soon as one hits generate it should update and the dynamic_panel_phenotypes changes?

> I suggest let's leave it as it is since it would just complicate the code without really adding much.

Should work though. 😆 There is always more to do; just drop it and add an issue to refactor it instead if you feel this is a good place to leave it!

Reply:
What about the phenotypes that have no genes associated?

Reply:
> > In the situation you describe is quite obvious that the panel has been modified. There are also these situations for dynamic panels created in the past, over which you have no control:
> > 
> > * Dynamic gene panels with genes added
> 
> Right, but the add gene endpoint does clear the `dynamic_panel_phenotypes` so that should be covered.
> 
> >

Unfortunately id doesn't: check the situation where you clear phenotypes and reset the dynamic gene line, then you add a gene manually:

<img width="1096" alt="image" src="https://user-images.githubusercontent.com/28093618/185396045-7f8f5a03-003e-4745-baa8-d38b063c39ef.png">

Reply:
😆 Ok, that is most likely also not supposed to happen: the user just removed the phenotypes and hit create hpo panel. The gene list is empty, which is fine, but the phenotype ids list on the dynamic panel reflects the old phenotypes on the case, not the current ones?!

Reply:
Fixing it. There is a problem if you try to export hpo panels but you have genes no phenotypes (it crashes)

Reply:
Tested on stage and works fine now. Merging!